src/test/java/ai/datahunters/md/server/server/PhotosEndpointTest.java

```diff
@@ -78,9 +78,9 @@ public void searchByType() throws IOException, JsonProcessingException {
 
     }
 
-    void compareBodies(String receivedResponse, String expectedResonse) {
+    void compareBodies(String receivedResponse, String expectedResponse) {
```

Comment:
If you are using similar method in your test suite, please extract it to a staitc method in a test utility class. Something like JsonUtils